src/v2/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx

```diff
@@ -24,18 +25,29 @@ const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
   // we still want to render the rest of the page.
   if (!hasFilter) return null
 
+  const sortOptions = [
+    { value: "-has_price,-prices", text: "Price (desc.)" },
+    { value: "-has_price,prices", text: "Price (asc.)" },
+    { value: "-partner_updated_at", text: "Recently updated" },
+    { value: "-published_at", text: "Recently added" },
+    { value: "-year", text: "Artwork year (desc.)" },
+    { value: "year", text: "Artwork year (asc.)" },
+  ]
+
+  const defaultSortValue =
+    sd["DECAYED_MERCH_V2"] === "experiment" ? "-year" : "-decayed_merch"
```

Comment:
I went with "-year" to enable an easier QA'ing of the experience (with the cookie/query param set to 'experiment', my 'Default' sort should be "-year", and the AB test should denotes the 'experiment' property in the AB test event. Set to 'control', it uses "-decayed_merch" and denotes the 'control' property in the AB test event).

Reply:
Prob don't need to use array access notation here for getting at the object

Reply:
Nice, so `-year` can just switch to `-decayed_merch_v2` once [this change](https://github.com/artsy/gravity/commit/a115b2b6e9aa97412febf2dbcda4d633c2975196#diff-047f9519cebd066e6fffb8f08372c8c5d72e166b01677ed81cc7a3b938632d26R177) is out

Reply:
Yep, exactly. Using `-decayed_merch_v2` results in a 400 error from Gravity (due to the validation in https://github.com/artsy/gravity/blob/b5dbe5549fdd84d606de6f133d282d09a12931e3/app/api/v1/filter_endpoint.rb#L45), so it was hard to test that specific value (artwork grid disappears and errors out).

There's a few places in this PR where that string needs to update when we're ready to launch the test (or at least QA the behavior/analytics on staging or in a review app).